### DISCUSSION TITLE
origami/2023/08/13/sorting-part-1

### DISCUSSION THREAD

**Discussion by giscus** [oojiang/oliverjiang.me-comments#1] [Announcements]
# origami/2023/08/13/sorting-part-1

Throughout junior and senior year of college, I would get these really delicious mooncake-like snacks from Costco called “Egg Yolk Pies.” The outside was flaky and crispy, and inside was a salty egg yolk surrounded by a sweet filling. They came packaged in tin boxes that stacked neatly on top of eachother, which I found perfect for storing origami. Instead of cluttering my desk and floor with origami, I just stuffed everything–the good, the bad, the incomplete–into egg yolk pie boxes, which I stacked in a corner of my room.

http://localhost:4000/origami/2023/08/13/sorting-part-1.html

**Comment by oojiang** [comment]
Part 2 here! ---> http://oliverjiang.me/origami/2023/12/19/sorting-part-2.html
